unit-tests/unit-test-config.py

```diff
@@ -174,14 +174,20 @@ def process_cpp( dir, builddir ):
         # We need the project name unique: keep the path but make it nicer:
         testname = 'test-' + testparent.replace( '/', '-' ) + '-' + os.path.basename(testdir)[5:]   # "test-log-internal-all"
 
+        config = libci.TestConfigFromCpp( dir + os.sep + f, context )
+        if f.startswith( "live/" ) != len(config.configurations) > 0:
```

Comment:
I decided to check the actual path and not the tags so that you couldn't add a `live` tag manually to override this